web/pom.xml

```diff
@@ -875,10 +841,221 @@
     </plugins>
   </build>
 
-  <!-- Define profiles for running configuration using -Denv=profile.id parameter
-    Profiles configuration is defined in src/main/filters.
-    -->
   <profiles>
+
+    <!-- Schema management profiles. -->
+    <profile>
+      <id>schema-iso19115-3-2018</id>
+      <activation>
+        <property><name>schemasCopy</name><value>!true</value></property>
+        <file><exists>../schemas/iso19115-3.2018</exists></file>
+      </activation>
+      <dependencies>
+        <dependency>
+          <groupId>${project.groupId}</groupId>
+          <artifactId>schema-iso19115-3.2018</artifactId>
+          <version>${gn.schemas.version.range}</version>
+        </dependency>
+      </dependencies>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>iso19115-3.2018-resources</id>
+                <phase>process-resources</phase>
+                <goals><goal>unpack</goal></goals>
+                <configuration>
+                  <encoding>UTF-8</encoding>
```

Comment:
Intellij give me an error on this line.  It indicates that encoding is not allowed here?  Based on the following 
https://maven.apache.org/plugins/maven-resources-plugin/examples/encoding.html the encoding is on a configuration element under plugin not execution.

Reply:
Thanks, that took some research to double check:
- Configuration is allowed in both cases
- My mistake is that `encoding` is not supported at all